core/src/main/scala/com/evolutiongaming/catshelper/SerialKey.scala

```diff
@@ -17,7 +17,7 @@ object SerialKey {
     def apply[A](key: K)(task: F[A]) = task.map { _.pure[F] }
   }
 
-  def of[F[_]: Concurrent: Parallel: Runtime, K: Hash]: F[SerialKey[F, K]] = {
```

Comment:
these are breaking changes and not relevant to original problem, could you please deprecate this method and create a new one as separate MR? as I can accept other part already :)

Reply:
Sure, I just no-brain fixed some warnings :)